tests/integration/gitlab/test_service.py

```diff
@@ -80,3 +82,8 @@ def test_project_create_in_the_group(self):
             repo=name_of_the_repo, namespace=namespace_of_the_repo
         )
         assert project.gitlab_repo
+
+    def test_service_without_auth(self):
+        service = GitlabService()
+        assert service.gitlab_instance
+        assert service.get_project(namespace="gnachman", repo="iterm2").exists()
```

Comment:
Why this?..;)

Reply:
Do we need to worry about the renaming of their repo? Or moving to `iterm2/iterm2` or something else?

Reply:
they're already at version 3, so I wouldn't worry about it... I took it from the most stars and it was shorter than `inkscape/inkscape` :)

Reply:
..;)